acceptance_tests/docker-compose.yaml

```diff
@@ -17,6 +17,23 @@ services:
       - acceptance_config
     user: www-data
 
+  qgis-landingage:
+    image: camptocamp/qgis-server:${DOCKER_TAG}
+    environment:
+      QGIS_SERVER_LOG_LEVEL: '0'
+      QGIS_CATCH_SEGV: '1'
+      QGIS_AUTH_DB_DIR_PATH: /etc/qgisserver/
+      PGSERVICEFILE: /etc/qgisserver/pg_service.conf
+      QGIS_SERVER_LANDING_PAGE_PREFIX: /mapserv_proxy/qgis
+      QGIS_SERVER_LANDING_PAGE_PROJECTS_DIRECTORIES: /etc/qgisserver
```

Comment:
Do we really need a new service for these 2 additional env vars ?

Reply:
May be yes because qgis has a QGIS_PROJECT_FILE

Reply:
Comment added :-)